src/drafts/Tooltip/Tooltip.tsx

```diff
@@ -273,9 +273,9 @@ export const Tooltip = React.forwardRef(
             React.cloneElement(child as React.ReactElement<TriggerPropsType>, {
               ref: triggerRef,
               // If it is a type description, we use tooltip to describe the trigger
-              'aria-describedby': type === 'description' ? `tooltip-${tooltipId}` : undefined,
+              'aria-describedby': type === 'label' ? `tooltip-${tooltipId}` : child.props['aria-describedby'],
```

Comment:
I think this PR provided a great opportunity to validate some of the implementation details! Thanks again for using Tooltip here 🙌🏻

I am not sure if updating the string `description` to `label` was intentional here but it unfortunately introduces an issue to the Tooltip. We need to make sure we render 'aria-describedby' for the description type tooltips and this line does that. So I think if we could update this line as 

```suggestion
              'aria-describedby': type === 'description' ? `tooltip-${tooltipId}` : child.props['aria-describedby'],
```

it will add `aria-describedby` attribute to the child element for description type, or use the child's existing `aria-describedby` attribute if there is one and not description type tooltip

With this update, current inactive story suffice the criteria Eric described [here](https://github.com/primer/react/pull/3913#discussion_r1388600092). Let me know if I am missing anything

Reply:
Oops - not intentional, just a copy/paste mistake. Your suggestion is what I meant 😅